src/PublicApiAnalyzers/Core/Analyzers/PublicApiAnalyzerResources.resx

```diff
@@ -145,6 +146,16 @@
   <data name="ShouldAnnotateApiFilesDescription" xml:space="preserve">
     <value>All public types and members should be declared with nullability annotations in PublicAPI.txt. This draws attention to API nullability changes in the code reviews and source control history, and helps prevent breaking changes.</value>
   </data>
+
+  <data name="ObliviousPublicApiTitle" xml:space="preserve">
+    <value>Public members should not use oblivious types</value>
+  </data>
+  <data name="ObliviousPublicApiMessage" xml:space="preserve">
+    <value>Symbol '{0}' uses some oblivious reference types.</value>
```

Comment:
We might find it helpful in the future to switch to this form:

```suggestion
    <value>Symbol '{0}' uses oblivious reference type '{1}'</value>
```

Reply:
I'd rather not. The tracking of which part or parts caused this warning would be tricky to do. Also, in practice, I'm not sure it's often ambiguous (either you put `#nullable enable` in the file or you didn't).